fastavro/_logical_writers.pyx

```diff
@@ -50,7 +53,17 @@ cpdef prepare_timestamp_millis(object data, schema):
             return mktime(& time_tuple) * MLS_PER_SECOND + <long64>(
                 int(data.microsecond) / 1000)
         else:
-            return <long64>(<double>(data.timestamp()) * MLS_PER_SECOND)
+            if is_windows:
+                if data.tzinfo is not None:
```

Comment:
Could we add a small comment here saying why we need to special case this with a link to the bug tracker? Something like this:

`On Windows, timestamps before the epoch will raise an error. See https://bugs.python.org/issue36439`